coala_quickstart/generation/FileGlobs.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+from coala_utils.Question import ask_question
+from coala_quickstart.Strings import GLOB_HELP
+from coalib.collecting.Collectors import collect_files
+
+
+def get_project_files(log_printer, printer, project_dir):
+    """
+    Gets the list of files matching files in the user's project directory
+    after prompting for glob expressions.
+
+    :param log_printer:
+        A ``LogPrinter`` object.
+    :param printer:
+        A ``ConsolePrinter`` object.
+    :return:
+        A list of file paths matching the files.
+    """
+    printer.print(GLOB_HELP)
+    file_globs = ask_question(
+        "Which files do you want coala to run on?",
+        default="**",
+        printer=printer,
+        typecast=list)
+    ignore_globs = ask_question(
+        "Which files do you want coala to run on?",
+        printer=printer,
+        typecast=list)
+    printer.print()
+
+    file_path_globs = [os.path.join(
+        project_dir, glob_exp) for glob_exp in file_globs]
```

Comment:
actually this is dangerous because the project_dir is not globbified, right? CC @fneu what did we do in this case at coala? I think we glob escaped the project_dir.